libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -245,14 +245,14 @@ InternalDispatchMap IRGenerator::generateInternalDispatchFunctions()
 	return internalDispatchMap;
 }
 
-string IRGenerator::generateFunction(FunctionDefinition const& _function)
+string IRGenerator::generateFunction(FunctionDefinition const& _function, bool _includeModifiers)
 {
-	string functionName = IRNames::function(_function);
+	string functionName = _includeModifiers ? IRNames::function(_function) : IRNames::functionWithModifierInner(_function);
 	return m_context.functionCollector().createFunction(functionName, [&]() {
-		solUnimplementedAssert(_function.modifiers().empty(), "Modifiers not implemented yet.");
```

Comment:
Modifiers in constructors are not implemented yet?

Reply:
They are now.